tests/modules/services/window-managers/sway/default.nix

```diff
@@ -2,6 +2,7 @@
   sway-bar-focused-colors = ./sway-bar-focused-colors.nix;
   sway-bindkeys-to-code = ./sway-bindkeys-to-code.nix;
   sway-default = ./sway-default.nix;
+  sway-extra-config = ./sway-extra-config.nix;
```

Comment:
Could you merge this with another test?
Adding another separate test slows down testing unfortunately.

Reply:
Ah, makes sense, done in 84bfccd9205267c31f23da099a84519aa3cd1372.